apps/docs/scripts/functions/generateSection.ts

```diff
@@ -210,6 +212,7 @@ export function generateSection(section: InputSection, articles: Articles, index
 		index,
 		categories,
 		content: '',
+		hero: section.hero ?? null,
```

Comment:
I don't remember why we did this, but I'm afraid to make it optional without digging into the original reason. Maybe because this used to be JSON serialized or something?